<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>remove_duplicate_images</class>
 <widget class="QWidget" name="remove_duplicate_images">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Folder</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditFolder"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonSelectFolder">
       <property name="text">
        <string>Select folder</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Threshold</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="horizontalSliderThreshold">
       <property name="value">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinBoxThreshold">
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxRecursive">
     <property name="text">
      <string>remove  image recursive</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelImageSize">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonRemove">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>86</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxThreshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderThreshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
